### ISSUE TITLE
<AP> LS475W User Manual Update

### ISSUE THREAD

**Issue by DOD-AP-Sales** [dod-taiwan/MISC#7]
Please amend the page 11 of the LS475W user manual to actual behavior: 15s+30s. The current UM says:
● SOS Emergency File Lock【 】
  While recording, press this button to lock the current recording file and protect it from being over written.

**Comment by dod-pm1** [comment]
???  you mean the icon is missing?
or you wish to add the actual file time?

**Comment by DOD-AP-Sales** [comment]
nope...we talked about this 13 days ago...the actual lock-file behavior is "15s + 30s" which is different from the UM says "lock the current recording file"....

**Comment by dod-pm1** [comment]
Oh that, sorry i got mixed up.
That, is in Marcom's hand, which they are recently caught up at loads of stuff and it seems like they are unable to provide a fast modification.
I will try to push them for sooner tho.
